File: Proyecto/es.efor.PryBase.Angular/webapp/src/app/shared/api/models/permission-dto.ts
/* tslint:disable */
import { DetailDto } from './detail-dto';
import { FunctionalityDto } from './functionality-dto';
import { UserDepartmentDto } from './user-department-dto';
import { UserLevelDto } from './user-level-dto';
export interface PermissionDto {
  department?: UserDepartmentDto;
  detail?: DetailDto;
  enabled?: boolean;
  functionality?: FunctionalityDto;
  id?: number;
  level?: UserLevelDto;
}

File: PryBase/es.efor.PryBase.Infraestructure/DTO/PermissionsDTOs/PermissionDTO.cs
using es.efor.PryBase.Infraestructure.DTO.UserDTOs;

namespace es.efor.PryBase.Infraestructure.DTO.PermissionsDTOs
{
    public class PermissionDTO
    {
        #region Constructor
        public PermissionDTO() { }
        #endregion

        #region Propiedades 
        public int Id { get; set; }
        public FunctionalityDTO Functionality { get; set; }
        public DetailDTO Detail { get; set; }
        public bool Enabled { get; set; }
        public UserDepartmentDTO Department { get; set; }
        public UserLevelDTO Level { get; set; }

        #endregion
    }
}
